doghouse/server/github_checker.go

```diff
@@ -20,9 +22,17 @@ type checkerGitHubClient struct {
 }
 
 func (c *checkerGitHubClient) GetPullRequestDiff(ctx context.Context, owner, repo string, number int) ([]byte, error) {
-	opt := github.RawOptions{Type: github.Diff}
-	d, _, err := c.PullRequests.GetRaw(ctx, owner, repo, number, opt)
-	return []byte(d), err
+	pr, _, err := c.PullRequests.Get(ctx, owner, repo, number)
+	if err != nil {
+		return nil, err
+	}
+
+	bytes, err := exec.Command("git", "diff", "--find-renames", pr.GetBase().GetSHA(), pr.GetHead().GetSHA()).CombinedOutput()
```

Comment:
Dogserver doesn't have git repository and it doesn't work.
Please revert the change.

Reply:
I revert it: https://github.com/reviewdog/reviewdog/pull/1697/commits/d72ab6bf3df88c3244619648365d7114a7a1af3d